core/src/main/java/io/grpc/internal/ServerCallImpl.java

```diff
@@ -205,7 +207,8 @@ public String getAuthority() {
    * on.
    */
   private void internalClose(Status internalError) {
-    stream.close(internalError, new Metadata());
+    log.log(Level.WARNING,format("cancelling the stream with status %s",internalError.toString()));
```

Comment:
Pass `internalError` as an argument and use MessageFormat-style syntax:
```java
log.log(Level.WARNING, "Cancelling the stream with status {0}", new Object[] {internalError});
```

We mainly use `String.format()` when we need to create a gRPC status description with an argument.